account_invoice_section_sale_order/models/sale_order.py

```diff
@@ -58,19 +57,17 @@ def _create_invoices(self, grouped=False, final=False, date=None):
                         )
                     )
                     sequence += 10
-                for move_line in self.env["account.move.line"].browse(move_line_ids):
-                    if move_line.display_type == "line_section":
-                        # add extra indent for existing SO Sections
-                        move_line.name = f"- {move_line.name}"
```

Comment:
Same comment applies as on #1871: Are you removing this extra indent bit on purpose? I don't see it in the original fixes that you are porting here, but then again it was introduced in the migration of v15 (https://github.com/OCA/account-invoicing/commit/439ff49ec071461344686f0bb9306a29ca4af9d3) and the fixes ported here are from 14.0.